doc/modules/impute.rst

```diff
@@ -133,12 +136,21 @@ Each of these 10 imputations is then put through the subsequent analysis pipelin
 (e.g. feature engineering, clustering, regression, classification). The 10 final
```

Comment:
I would remove the example of `10` and instead replace by `n` on the 2 lines above.